receipt_dynamo/tests/integration/test__lock_manager_production_scenario.py

```
@@ -0,0 +1,283 @@
+            
+            signal.signal(signal.SIGALRM, timeout_handler)
+            signal.alarm(10)
+            
+            try:
+                start = time.time()
+                result = manager.validate_ownership()
+                elapsed = time.time() - start
+                signal.alarm(0)
+                results[manager_name] = result
+                print(f"{manager_name} validate_ownership() completed in {elapsed:.2f}s")
+            except TimeoutError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)